docs/source/packages/implementations/release-notes/daml-finance-holding.rst

```diff
@@ -0,0 +1,13 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Daml.Finance.Holding - Release Notes
+####################################
+
+Version 1.0.1
```

Comment:
Should we have version `1.0.0` documented ?

Reply:
I see these release notes as "what has changed from the previous version". In the case of `1.0.0` packages, there is no previous version to compare with :)

Reply:
We have all the `0.x.x` packages that are still sitting in GitHub releases...

Reply:
Right, I should have reformulated with

> I see these release notes as "what has changed from the previous version". In the case of `1.0.0` packages, there is no previous version to compare with which is supported and for which we adhere to semantic versioning 😄

Reply:
Shall we rename "Release Notes" to "Changelog" to stress that this summarises changes compared to a previous version?

Reply:
> Shall we rename "Release Notes" to "Changelog" to stress that this summarises changes compared to a previous version?

I view a changelog as low level description of changes aimed at developers, whereas release notes are a higher level description of changes aimed at non-developers. So I would say renaming depends on whom your target audience is.

Reply:
I think renaming to "Changelog" makes sense. Release notes we probably only provide for SDK releases, and they can be extracted from the changelog. The changelog serves as a detailed per-package record of all changes between versions.

Reply:
👍 renamed